pkg/controller/plan/adapter/vsphere/builder.go

```diff
@@ -237,9 +237,6 @@ func (r *Builder) PodEnvironment(vmRef ref.Ref, sourceSecret *core.Secret) (env
 }
 
 func (r *Builder) mapMacStaticIps(vm *model.VM) string {
-	if !isWindows(vm) {
```

Comment:
we also need to remove [the check in the validator](https://github.com/kubev2v/forklift/blob/254f0093b0771b6177b50005ba0e08030bd9ccdc/pkg/controller/plan/adapter/vsphere/validator.go#L142) and remove the `isWindows` function then

Reply:
Owww, thanks, missed that

Reply:
on a second thought, if you're not going to use this for RHEL VMs, why do we need to pass this information to the conversion pod?..

Reply:
@liranr23 is using this env var to indicate `PreserveStaticIPs` , the ceck is:
https://github.com/kubev2v/forklift/blob/5010fb4b00e51fce6102ed0c2b02fc2dab2c398c/virt-v2v/cold/entrypoint.go#L131

Reply:
I can add an explicit `PreserveStaticIPs` but I think @liranr23 way is safer because he also check that the VM actually set the IPs manually and not just ask for static IPs

Reply:
i'm ok with adding the `isWindows` and the check in the validator. it's true we check if that's not an empty string in virt-v2v. it makes a bit nonsense to send it but if that help and covers all scenarios that's ok.

Reply:
> I can add an explicit `PreserveStaticIPs` but I think @liranr23 way is safer because he also check that the VM actually set the IPs manually and not just ask for static IPs

yeah but on the other hand we don't always know about that information - that's why we added that validation



> i'm ok with adding the `isWindows` and the check in the validator. it's true we check if that's not an empty string in virt-v2v. it makes a bit nonsense to send it but if that help and covers all scenarios that's ok.

I guess you meant `PreserveStaticIPs`, right?